includes/class-wc-geolite-integration.php

```diff
@@ -38,10 +38,6 @@ class WC_Geolite_Integration {
 	 */
 	public function __construct( $database ) {
 		$this->database = $database;
-
-		if ( ! class_exists( 'MaxMind\\Db\\Reader', false ) ) {
```

Comment:
We could move/copy `get_country_iso()` method to `WC_MaxMind_Geolocation_Database` or into the integration class, and deprecate this class.

Reply:
Agreed!